lib/s3middleware/azureHelpers/mpuUtils.js

```diff
@@ -111,12 +111,15 @@ log, cb) => {
         if (err) {
             log.error('Error from Azure data backend uploadPart',
                 { error: err.message, dataStoreName });
+            if (err.code === 'InvalidMd5') {
```

Comment:
@LaurenSpiegel should we consider porting the translate errors to Arsenal?
If someone deletes the container/bucket (although I think we should return `MpuInProgress` if someone tried that from ScalityConnect) before we get to this call, we would respond InternalError instead of NoSuchBucket for this call.

Reply:
Let's discuss tomorrow. I don't fully understand what you mean.